src/apps/media/src/app/MediaApp.js

```diff
@@ -221,6 +221,7 @@ export default connect((state) => {
               showDeleteGroupModal={showDeleteGroupModal}
               modal={props.modal}
               setCurrentGroupID={setCurrentGroupID}
+              groupID={props.groupID}
```

Comment:
You are passing a new Prop chain for groupID but I believe currentGroup already has the ID you need

line 55 of this file states:
```
const [currentGroupID, setCurrentGroupID] = useState(
    params.groupID || props.groupID
  );
```
along with a useEffect that updates the state whenever props.groupId changes

With this info you should be able remove the whole new prop chain you are adding since it looks like its already in place